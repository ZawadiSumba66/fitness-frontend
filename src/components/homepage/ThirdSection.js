import { Link } from '@reach/router';
import third from '../../assets/third.jpg';

const ThirdSection = () => (
  <div className=" my-5 py-5 third_section_wrapper">
    <div className="row align-items-center">
      <div className=" col-md-6">
        <img
          src={third}
          alt="FitHomeTwo"
          className="third_image"
        />
      </div>
      <div className="col-md-6">
        <div>
          <h2 className="text-lg-black font-weight-bold">
            We are here to guide you in every step.
          </h2>
          <p className="text-sm-gray mb-5 mt-3">
            The advice given by our experts is just enough to get you through the day.
            In the morning, evening, at home or at the gym. It does not matter. All we
            encourage is to start and see your dream healthy lifestyle come to a reality.
          </p>
          <Link to="/signup" className="button-orange text-light font-weight-bold px-4 py-2 mt-4">
            GET STARTED
          </Link>
        </div>
      </div>
    </div>
  </div>
);

export default ThirdSection;
