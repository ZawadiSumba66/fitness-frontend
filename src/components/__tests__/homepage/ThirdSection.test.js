import renderer from 'react-test-renderer';
import ThirdSection from '../../homepage/ThirdSection';

it('matches the third section snapshot', () => {
  const tree = renderer.create(<ThirdSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
