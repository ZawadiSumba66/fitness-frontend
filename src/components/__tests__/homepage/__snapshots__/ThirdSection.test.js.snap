// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the third section snapshot 1`] = `
<div
  className=" my-5 py-5 third_section_wrapper"
>
  <div
    className="row align-items-center"
  >
    <div
      className=" col-md-6"
    >
      <img
        alt="FitHomeTwo"
        className="third_image"
        src="third.jpg"
      />
    </div>
    <div
      className="col-md-6"
    >
      <div>
        <h2
          className="text-lg-black font-weight-bold"
        >
          We are here to guide you in every step.
        </h2>
        <p
          className="text-sm-gray mb-5 mt-3"
        >
          The advice given by our experts is just enough to get you through the day. In the morning, evening, at home or at the gym. It does not matter. All we encourage is to start and see your dream healthy lifestyle come to a reality.
        </p>
        <a
          className="button-orange text-light font-weight-bold px-4 py-2 mt-4"
          href="/signup"
          onClick={[Function]}
        >
          GET STARTED
        </a>
      </div>
    </div>
  </div>
</div>
`;
